docs/getting-started/flashing-firmware/flashing-external-serial-adapter.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: flashing-external-serial-adapter
+title: Flashing ESP32 Devices with an External Serial Adapter
+sidebar_label: Flashing with External Serial Adapter - ESP32 Devices
+sidebar_position: 2
```

Comment:
Not sure what sidebar position we want for this one, position 1 probably makes sense, though we may need to bump the other `docs/getting-started/flashing-firmware/*` files numbers. I've been trying to use the `pagination_prev` & `pagination_next` to make the "Next" buttons walk the user through their workflow. I'm always pointing users to Meshtastic-flasher. So for this page I'd add the following:
```
pagination_prev: getting-started/flashing-firmware/flashing-firmware
pagination_next: getting-started/flashing-firmware/meshtastic-flasher
```

I think `docs/getting-started/flashing-firmware/index.mdx` should bypass this page as this is kind of an "advanced" use case. So on index.mdx we can add:
```
pagination_next: getting-started/flashing-firmware/meshtastic-flasher
```
and on `meshtastic-flasher.mdx` we can add
```
pagination_prev: getting-started/flashing-firmware/flashing-firmware
```